editor/node_dock.cpp

```diff
@@ -70,6 +70,9 @@ void NodeDock::update_lists() {
 void NodeDock::set_node(Node *p_node) {
 	connections->set_node(p_node);
 	groups->set_current(p_node);
+	if (p_node) {
+		prev_node = p_node;
+	}
```

Comment:
So if I understand correctly, this can be better described as "last valid node" rather than "previous node", correct?

I suppose it works for the problem, but maybe we should at least rename the property (and the method)?